src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  padding-right: 16px;
+  padding-left: 15px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: white;
+  box-sizing: border-box;
+  text-align: center;
+  margin: 0 auto;
+}
+
+.imac img {
+  margin-top: 32px;
```

Comment:
Be consistent with your margins (Add only top or bottom, but not both) to avoid margin collapse